projects/argocd/components/kargo-secrets/kargo-secrets.cue

```diff
@@ -0,0 +1,70 @@
+package holos
+
+// Produce a helm chart build plan.
+holos: Component.BuildPlan
+
+Component: #Kubernetes & {
+	Resources: [_]: [_]: metadata: namespace: Kargo.Namespace
+
+	Resources: {
+		ExternalSecret: creds: {
+			metadata: name: Kargo.Values.api.secret.name
+			spec: {
+				refreshInterval: "24h"
+				target: {
+					creationPolicy: "Owner"
+					deletionPolicy: "Delete"
+					template: {
+						type:          "Opaque"
+						mergePolicy:   "Merge"
+						engineVersion: "v2"
+						data: ADMIN_ACCOUNT_PASSWORD_HASH: "{{ .password | bcrypt }}"
```

Comment:
@glarizza @natemccurdy FYI, here's how to generate passwords and bcrypt hashes using External Secrets Operator instead of a Job.  Works pretty well, but not super well documented.